src/feed/TickHistoryFormatter.js

```diff
@@ -1,16 +1,21 @@
 export class TickHistoryFormatter {
+    static getUTCDate = function (date) {
+        const UTCdate = date.toISOString();
+        return UTCdate.substring(0, 10).concat(' ', UTCdate.substring(11, 19));
```

Comment:
this can be simplified to `UTCdate.substring(0, 19)`. You only need to remove the `Z` at the back, which marks this as UTC time, causing ChartIQ to offset the timezone.

Reply:
@brucebinary done!